.tekton/scanner-pull-request.yaml

```diff
@@ -0,0 +1,446 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stackrox/scanner?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && source_branch.contains("rhtap")
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: acs
+    appstudio.openshift.io/component: scanner
+    pipelines.appstudio.openshift.io/type: build
+  name: scanner-on-pull-request
+  namespace: rh-acs-tenant
+spec:
+  params:
```

Comment:
[not only about these lines]
Could you please reformat these `.tekton` files as done in stackrox/stacrkox, [roxctl](https://github.com/stackrox/stackrox/blob/8510071fb1dfa95026f4bc4d871e29afec6a3bda/.tekton/roxctl-pull-request.yaml#L29-L29) for example?

When there are blank lines between blocks and certain blocks are re-ordered, reading and reviewing the pipeline becomes much easier.